productivity-suite/app/fragments/news/src/routes/[...news].tsx

```diff
@@ -1,18 +1,30 @@
 import {
   Component,
+  createEffect,
   createResource,
   createSignal,
   For,
   Show,
   Suspense,
 } from "solid-js";
+import { useSearchParams } from "solid-start";
 import Story from "~/components/story";
 import fetchAPI from "~/lib/api";
 import { IStory } from "~/types";
+import { getBus } from "piercing-library";
 
 export default function News() {
   const [page, setPage] = createSignal<number>(1);
 
+  const [searchParams] = useSearchParams<{ page?: string }>();
```

Comment:
Should the fragment be reading the search params?
I feel like it should be reading and writing this on the message bus?

Reply:
yeah I fell in the same mistake I did for the todos 🤦 

sorry about that